common/djangoapps/student/models.py

```diff
@@ -434,14 +436,15 @@ class Meta(object):
     phone = models.CharField(max_length=20, null=True, blank=True)
     second_name = models.CharField(max_length=100, null=True, blank=True)
     additional_email = models.EmailField(max_length=255, null=True, blank=True)
-    date_of_birth = models.DateField(blank=True, null=True, db_index=True)
+    date_of_birth = models.DateField(blank=True, null=True, db_index=True, validators=[validate_date_of_birth])
```

Comment:
I see that you add the validator directly on the model. Do not we have form to validate there?

Reply:
Yes, we can. I thought that will be a good idea.

Reply:
Done